app/scripts/AddTrackPositionMenu.jsx

```diff
@@ -4,56 +4,56 @@ import React from 'react';
 import withTheme from './hocs/with-theme';
 import { THEME_DARK } from './configs';
 
-import '../styles/AddTrackPositionMenu.module.scss';
+import classes from '../styles/AddTrackPositionMenu.module.scss';
 
 function AddTrackPositionMenu(props) {
-  let tableStyleNames = 'add-track-position-table';
+  const tableClassNames = [classes['add-track-position-table']];
   if (props.theme === THEME_DARK)
-    tableStyleNames += ' add-track-position-table-dark';
+    tableClassNames.push(classes['add-track-position-table-dark']);
   return (
     <div>
-      <div styleName="add-track-position-span">Add Track...</div>
-      <table styleName={tableStyleNames}>
+      <div className={classes["add-track-position-span"]}>Add Track...</div>
+      <table className={tableClassNames.join(" ")}>
```

Comment:
We could add a tiny runtime dependency like [`classnames`](https://www.npmjs.com/package/classnames) to make this a little less clunky, e.g.,

```javascript
const className = classNames(classes['add-track-position-table'], {
	[classes['add-track-position-table-dark']]: props.theme === THEME_DARK,
})
```

Reply:
Update: I added the tiny [`clsx`](https://www.npmjs.com/package/clsx) as a dependency for this purpose.

Reply:
Update, I added [`clsx`](https://github.com/lukeed/clsx) instead.